apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -85,7 +85,9 @@ public function register_scripts() {
 			plugins_url( 'dist/starter-page-templates.min.js', __FILE__ ),
 			array( 'wp-plugins', 'wp-edit-post', 'wp-element' ),
 			filemtime( plugin_dir_path( __FILE__ ) . 'dist/starter-page-templates.min.js' ),
-			true
+			array(
+				'strategy' => 'defer',
+			)
```

Comment:
Turns out there is a backwards compatible way that goes like this:

```suggestion
 	true 
 );
 		wp_script_add_data( 'starter-page-templates', 'strategy', 'defer' );

```